multiapi-engine/pom.xml

```diff
@@ -24,6 +24,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-web</artifactId>
+      <version>5.3.19</version>
+    </dependency>
```

Comment:
I think add a new dependency just to use the `RequestMethod.values` method which only return a list of enums with the http operations not the best solution, I could suggest to you create your own class enum for this purpose. 

And by the way be careful when you add a new dependency, because in this case this version of spring-web have a known vulnerability